src/pages/guidelines/spacing.mdx

```diff
@@ -76,85 +72,166 @@ Below is the example of a regular Content block simple on the left and how the s
+| `$spacing-08` | 2.5   | 40  | <div style={{width: '40px', height: '40px', background: '#A56EFF'}}></div>   |
+| `$spacing-09` | 3     | 48  | <div style={{width: '48px', height: '48px', background: '#A56EFF'}}></div>   |
+| `$spacing-10` | 4     | 64  | <div style={{width: '64px', height: '64px', background: '#A56EFF'}}></div>   |
+| `$spacing-11` | 5     | 80  | <div style={{width: '80px', height: '80px', background: '#A56EFF'}}></div>   |
+| `$spacing-12` | 6     | 96  | <div style={{width: '96px', height: '96px', background: '#A56EFF'}}></div>   |
+| `$spacing-13` | 10    | 160 | <div style={{width: '160px', height: '160px', background: '#A56EFF'}}></div> |
 
 </div>
 
 <br />
 
-## Content component spacing patterns
+## Spacing concepts
+
+### Internal vs external spacing
+
+Components containing multiple elements may have internal spacing to help guide how those elements display as the component resizes across breakpoints.
+
+#### Internal spacing
+
+The internal spacing for card determines how the various elements are spaced within the container, including when some elements are visible or hidden.
+
+<Row>
+<Column colLg={8}>
+
+![A card component with internal spacing highlighted](../../images/guidelines/spacing/guidelines-spacing-spacing-concepts-01.png)
+
+</Column>
+</Row>
+
+#### External spacing
+
+The external spacing for a content component like content block determines how the component will merge spacing with other content components. Because lead space and footer do not have margin external spacing, the content components will provide the proper spacing to start and end the page.
+
+<Row>
+<Column colLg={8}>
+
+![A content block component with external spacing highlighted](../../images/guidelines/spacing/guidelines-spacing-spacing-concepts-02.png)
+
+</Column>
+</Row>
+
+External spacing defines the area outside the component (usually vertically). This spacing can be important when thinking about how the component will nest inside other components, or what happens when it appears directly on top of another component. It is important to consider the spacing value as well as the type ([margin or padding](#margin-vs-padding)). If you are contributing a new component to Carbon for IBM.com, consider how your component will fit with others when it appears on the page.
+
+### Margin vs padding
+
+Beyond spacing values, it is is important to understand spacing types. Margin and padding are used purposefully to ensure components fit together seamlessly. When a component is viewed on its own, its external spacing may appear the same whether margin or padding is used. However, a component will behave very differently if it is nested within another component or when it is placed after other components. This is because padding will preserve space and stack one component on top of the other, while margin wil allow spacing to merge together while keeping the largest spacing value.
```

Comment:
```suggestion
Beyond spacing values, it is is important to understand spacing types. Margin and padding are used purposefully to ensure components fit together seamlessly. When a component is viewed on its own, its external spacing may appear the same whether margin or padding is used. However, a component will behave very differently if it is nested within another component or when it is placed after other components. This is because padding will preserve space and stack one component on top of the other, while margin will allow spacing to merge together while keeping the largest spacing value.
```